### PR TITLE
Stop re-publishing deprecated packages with each new release

### PR DESCRIPTION
**Pull request by CharliePoole**
Part of #720 - more work needed post 3.13.

### CODE REVIEW

File: build.cake [hunk 1]
```diff
@@ -482,17 +478,17 @@ Task("BuildChocolateyPackages")
...
+        //    new ChocolateyPackSettings()
+        //    {
+        //        Version = productVersion,
+        //        OutputDirectory = PACKAGE_DIR,
+        //        Files = new [] {
+        //            new ChocolateyNuSpecContent { Source = CURRENT_IMG_DIR + "LICENSE.txt", Target = "tools" },
+        //            new ChocolateyNuSpecContent { Source = CURRENT_IMG_DIR + "NOTICES.txt", Target = "tools" },
+        //            new ChocolateyNuSpecContent { Source = CHOCO_DIR + "VERIFICATION.txt", Target = "tools" }
+        //        }
+        //    });
```
Comment (mikkelbu) [inline build.cake:491]: Should this be removed?

File: package-checks.cake [hunk 2]
```diff
@@ -78,17 +78,17 @@ public void CheckAllPackages()
             "NUnit.Engine.Api",
             HasDirectory("lib/net20").WithFile("nunit.engine.api.pdb"),
             HasDirectory("lib/netstandard2.0").WithFile("nunit.engine.api.pdb")) &
-        CheckNuGetPackage(
-            "NUnit.Runners",
-            HasFile("LICENSE.txt")) &
+        //CheckNuGetPackage(
+        //    "NUnit.Runners",
+        //    HasFile("LICENSE.txt")) &
```
Comment (mikkelbu) [inline package-checks.cake:83]: Should this be removed?

File: package-checks.cake [hunk 3]
```diff
@@ -78,17 +78,17 @@ public void CheckAllPackages()
...
             "nunit-console-runner",
             HasDirectory("tools").WithFiles("LICENSE.txt", "NOTICES.txt", "VERIFICATION.txt").AndFiles(CONSOLE_FILES).AndFiles(ENGINE_FILES).AndFile("nunit.choco.addins"),
             HasDirectory("tools/agents/net20").WithFiles(AGENT_FILES).AndFile("nunit.agent.addins"),
             HasDirectory("tools/agents/net40").WithFiles(AGENT_FILES).AndFile("nunit.agent.addins")) &
-        CheckChocolateyPackage(
-            "nunit-console-with-extensions",
-            HasDirectory("tools").WithFiles("LICENSE.txt", "NOTICES.txt", "VERIFICATION.txt")) &
+        //CheckChocolateyPackage(
+        //    "nunit-console-with-extensions",
+        //    HasDirectory("tools").WithFiles("LICENSE.txt", "NOTICES.txt", "VERIFICATION.txt")) &
```
Comment (mikkelbu) [inline package-checks.cake:91]: Same as above

**Review by mikkelbu — APPROVED** [review]
Just a couple of comments regarding if we should delete the code that has just been commented out.

File: build.cake (hunk 1 above)
Comment (CharliePoole) [reply build.cake:491]: Definitely... I'm afraid I'm jumping back and forth across too many projects and missing stuff!!!

File: package-checks.cake (hunk 2 above)
Comment (CharliePoole) [reply package-checks.cake:83]: Ditto

**Pull request merged**